config/locales/en/shared/schools/school_contact_details.yml

```
@@ -0,0 +1,10 @@
+en:
+  shared:
+    schools:
+      school_contact_details:
+        no_placement_coordinator: This organisation does not have a school contact.
+        add_placement_coordinator: Add school contact
+        full_name: Full name
+        email: Email
+        change: Change
+        remove_placement_coordinator: Remove school contact
```

We should probably rename these keys so they match the new name for this feature.

```suggestion
        no_school_contact: This organisation does not have a school contact.
        add_school_contact: Add school contact
        full_name: Full name
        email: Email
        change: Change
        remove_school_contact: Remove school contact
```